### PR TITLE
Fixes and modifications to classes NXapm_paraprobe_* classes

### PR DESCRIPTION
**Pull request by mkuehbach**
This PR implements changes and fixes brought about by using the NXapm_paraprobe_*-specific NeXus classes in the paraprobe-toolbox. Paraprobe-toolbox is the reference implementation to show how NeXus can be used in the atom probe community

https://gitlab.com/paraprobe/paraprobe-toolbox/-/tree/composing-v0.5?ref_type=heads is the respective feature branch of the paraprobe-toolbox scientific software that implements that usage of NeXus for atom probe.
Adding NeXus APMv2 consistently throughout the tool is the final piece of complete v0.5 of the toolbox and summarize that work which took place in the FAIRmat project.

### CODE REVIEW

File: contributed_definitions/nyaml/NXapm_paraprobe_config_distancer.yaml [hunk 1]
```diff
@@ -1,453 +0,0 @@
-category: application
```
Comment (sanbrock) [inline contributed_definitions/nyaml/NXapm_paraprobe_config_distancer.yaml:1]: github shows a removed file and a new one. Maybe a renaming step would have helped in reviewing the changes.

**Review by sanbrock — CHANGES_REQUESTED** [review]

File: contributed_definitions/nyaml/NXapm_paraprobe_config_distancer.yaml (hunk 1 above)
Comment (mkuehbach) [reply contributed_definitions/nyaml/NXapm_paraprobe_config_distancer.yaml:1]: Agree, I should have made a commit where I just renamed and making edits thereafter

File: contributed_definitions/nyaml/NXms_score_config.yaml [hunk 2]
```diff
@@ -11,12 +11,10 @@ symbols:
 type: group
 NXms_score_config(NXobject):
   (NXentry):
-    \@version:
-      doc: |
-        Version specifier of this application definition.
-    definition:
+    definition(NX_CHAR):
       doc: |
         Official NeXus NXDL schema with which this file was written.
```
[the remark this answers is not in the document]
Comment (mkuehbach) [reply contributed_definitions/nyaml/NXms_score_config.yaml:16]: Fixed

File: contributed_definitions/nyaml/NXms_score_results.yaml [hunk 3]
```diff
@@ -35,15 +35,10 @@ symbols:
...
-    \@version:
-      doc: |
-        An at least as strong as SHA256 hashvalue of the file
-        that specifies the application definition.
-    
-    # enumeration: [sha_256_hash]
-    definition:
+    definition(NX_CHAR):
       doc: |
         NeXus NXDL schema to which this file conforms.
```
[the remark this answers is not in the document]
Comment (mkuehbach) [reply contributed_definitions/nyaml/NXms_score_results.yaml:40]: fixed

**Review by sanbrock — COMMENTED** [review]
LGTM

**Pull request merged**